libs/barista-components/quick-filter/src/quick-filter.spec.ts

```diff
@@ -194,6 +234,20 @@ describe('dt-quick-filter', () => {
       subscription.unsubscribe();
     }));
 
+    it('should propagate sidebarOpenChange event when emitted by the drawer', (done) => {
+      const spy = jest.fn();
+      const subscription = quickFilterInstance.sidebarOpenChange.subscribe(spy);
+      zone.simulateZoneExit();
+
+      drawerInstance.openChange.emit(true);
+      setTimeout(() => {
```

Comment:
Please avoid setTimeout here. A `fixture.detectChanges` or `zone.simulateZoneExit()` should be good enough. SimulateZoneExit possibly only works in a fakeAsync test.

Reply:
The problem is connected with the fact that the drawer EventEmitter is configured to emit events in async.
But I think I can actually solve the issue using `tick()` to force the async event through.